buildspec-dotnetcore3.1.yml

```diff
@@ -0,0 +1,18 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      dotnet: 3.1
+
+    commands:
+      - pip install --upgrade pip
+      - pip install -r requirements.txt
+      - echo 'export PATH="$PATH:/root/.dotnet/tools"' >> ~/.bash_profile
```

Comment:
is dotnet not on the path by default?

Reply:
Do the commands not run in the same process or sub-processes? If yes, we can bypass `.bash_profile` and `export` to the shell directly.

Reply:
Good point. After I pass `env` to `Popen`, it seems to work out of the box.